src/modules/dashboard/lists/BorrowAssetsList/BorrowAssetsListMobileItem.tsx

```diff
@@ -73,19 +73,15 @@ export const BorrowAssetsListMobileItem = ({
         captionVariant="description"
         mb={2}
       >
-        <Box sx={{ display: 'flex', flexDirection: 'column' }}>
-          <IncentivesCard
-            value={Number(variableBorrowRate)}
-            incentives={vIncentivesData}
-            symbol={symbol}
-            variant="secondary14"
-          />
-          <MeritIncentivesButton
-            symbol={symbol}
-            market={currentMarket}
-            protocolAction={ProtocolAction.borrow}
-          />
-        </Box>
+        <IncentivesCard
+          value={Number(variableBorrowRate)}
+          incentives={vIncentivesData}
+          address={variableDebtTokenAddress}
```

Comment:
this is only used for the zksync incentives right?

Reply:
Yes exactly